src/ResourceManager/Resources/Commands.Resources/RoleAssignments/GetAzureRoleAssignmentCommand.cs

```diff
@@ -36,6 +36,7 @@ public class GetAzureRoleAssignmentCommand : ResourcesBaseCmdlet
         [Parameter(Mandatory = true, ValueFromPipelineByPropertyName = true, ParameterSetName = ParameterSet.ScopeWithObjectId,
             HelpMessage = "The user or group object id.")]
         [ValidateNotNullOrEmpty]
+        [Alias("Id", "PrincipalId")]
```

Comment:
Why adding `PrincipalId` here? This is the ID for the AD object regardless of its type